pocs/observatory.py

```diff
@@ -47,10 +47,15 @@ def __init__(self, *args, **kwargs):
         self.mount = None
         self._create_mount()
 
-        self.logger.info('\tSetting up cameras')
-        self.cameras = OrderedDict()
-        self._primary_camera = None
-        self._create_cameras(**kwargs)
+        if not cameras:
+            cameras = OrderedDict()
```

Comment:
Why not just raise an exception?

Reply:
I could. `create_dome_from_config` is returning None so I'll go for consistency with that (although see comment below).